src/targets/gpu/fuse_mlir.cpp

```diff
@@ -79,11 +81,23 @@ static bool is_negated_op(const std::string& s)
     return contains({'!', '~'}, s[0]);
 }
 
+#ifndef MIGRAPHX_USE_ROCBLAS
+const std::vector<std::string>& get_default_options()
+{
+    static std::vector<std::string> default_options{"dot", "convolution", "fused"};
+    return default_options;
+}
+#endif
```

Comment:
We should not be changing how the environment variable works. This also disables miopen convolutions as well, which seems unrelated.